iocore/net/P_SSLNetVConnection.h

```diff
@@ -295,7 +302,7 @@ class SSLNetVConnection : public UnixNetVConnection
     HANDSHAKE_HOOKS_DONE
   } sslHandshakeHookState;
 
-  const SSLNextProtocolSet *npnSet;
+  mutable SSLNextProtocolSet *npnSet;
```

Comment:
Does this need to be `mutable`? Where does a `const SSLNetVConnection` have its NPN set modified?